[BUG] fix ut

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

The default clean mode is changed from `onDemand` to `onRuntimeDeleted`

```
2021/12/15 02:57:47 Using CRITICAL_FUSE_POD = false
2021/12/15 02:57:47 Using INIT_PORT_CHECK_ENABLED = false
--- FAIL: TestCleanPolicy (0.03s)
    --- FAIL: TestCleanPolicy/default_test-juicefs (0.01s)
        runtime_test.go:518: GetRuntimeInfo() = &base.RuntimeInfo{name:"default_policy_juicefs", namespace:"default", runtimeType:"juicefs", tieredstoreInfo:base.TieredStoreInfo{Levels:[]base.Level(nil)}, exclusive:false, fuse:base.Fuse{Global:true, NodeSelector:map[string]string(nil), CleanPolicy:"OnRuntimeDeleted"}, deprecatedNodeLabel:false, deprecatedPVName:false}, want &base.RuntimeInfo{name:"default_policy_juicefs", namespace:"default", runtimeType:"juicefs", tieredstoreInfo:base.TieredStoreInfo{Levels:[]base.Level(nil)}, exclusive:false, fuse:base.Fuse{Global:true, NodeSelector:map[string]string(nil), CleanPolicy:"OnDemand"}, deprecatedNodeLabel:false, deprecatedPVName:false}
--- FAIL: TestGetRuntimeInfo (0.01s)
    --- FAIL: TestGetRuntimeInfo/juicefs_test (0.00s)
        runtime_test.go:745: GetRuntimeInfo() = &base.RuntimeInfo{name:"juice", namespace:"default", runtimeType:"juicefs", tieredstoreInfo:base.TieredStoreInfo{Levels:[]base.Level(nil)}, exclusive:false, fuse:base.Fuse{Global:true, NodeSelector:map[string]string(nil), CleanPolicy:"OnRuntimeDeleted"}, deprecatedNodeLabel:false, deprecatedPVName:false}, want &base.RuntimeInfo{name:"juice", namespace:"default", runtimeType:"juicefs", tieredstoreInfo:base.TieredStoreInfo{Levels:[]base.Level(nil)}, exclusive:false, fuse:base.Fuse{Global:true, NodeSelector:map[string]string(nil), CleanPolicy:"OnDemand"}, deprecatedNodeLabel:false, deprecatedPVName:false}
Running Suite: Base Suite
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/charts/fluid/fluid/Chart.yaml
+++ b/charts/fluid/fluid/Chart.yaml
@@ -18,7 +18,7 @@ version: 0.7.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 0.7.0-e3fde3d
+appVersion: 0.7.0-d06954c
 home: https://github.com/fluid-cloudnative/fluid
 keywords:
   - category:data

--- a/charts/fluid/fluid/values.yaml
+++ b/charts/fluid/fluid/values.yaml
@@ -6,7 +6,7 @@ workdir: /tmp
 
 dataset:
   controller:
-    image: fluidcloudnative/dataset-controller:v0.7.0-e3fde3d
+    image: fluidcloudnative/dataset-controller:v0.7.0-d06954c
 
 csi:
   config:
@@ -14,7 +14,7 @@ csi:
   registrar:
     image: registry.aliyuncs.com/acs/csi-node-driver-registrar:v1.2.0
   plugins:
-    image: fluidcloudnative/fluid-csi:v0.7.0-e3fde3d
+    image: fluidcloudnative/fluid-csi:v0.7.0-d06954c
   kubelet:
     rootDir: /var/lib/kubelet
 
@@ -27,9 +27,9 @@ runtime:
     portRange: 20000-26000
     enabled: true
     init:
-      image: fluidcloudnative/init-users:v0.7.0-e3fde3d
+      image: fluidcloudnative/init-users:v0.7.0-d06954c
     controller:
-      image: fluidcloudnative/alluxioruntime-controller:v0.7.0-e3fde3d
+      image: fluidcloudnative/alluxioruntime-controller:v0.7.0-d06954c
     runtime:
       image: registry.aliyuncs.com/alluxio/alluxio:release-2.7.0-SNAPSHOT-c058736
     fuse:
@@ -43,19 +43,19 @@ runtime:
     fuse:
       image: registry.cn-shanghai.aliyuncs.com/jindofs/jindo-fuse:3.7.3
     controller:
-      image: fluidcloudnative/jindoruntime-controller:v0.7.0-e3fde3d
+      image: fluidcloudnative/jindoruntime-controller:v0.7.0-d06954c
     init:
       portCheck:
         enabled: false
-      image: fluidcloudnative/init-users:v0.7.0-e3fde3d
+      image: fluidcloudnative/init-users:v0.7.0-d06954c
   goosefs:
     runtimeWorkers: 3
     portRange: 26000-32000
     enabled: false
     init:
-      image: fluidcloudnative/init-users:v0.7.0-e3fde3d
+      image: fluidcloudnative/init-users:v0.7.0-d06954c
     controller:
-      image: fluidcloudnative/goosefsruntime-controller:v0.7.0-e3fde3d
+      image: fluidcloudnative/goosefsruntime-controller:v0.7.0-d06954c
     runtime:
       image: ccr.ccs.tencentyun.com/qcloud/goosefs:v1.1.0 
     fuse:
@@ -63,11 +63,11 @@ runtime:
   juicefs:
     enabled: false
     controller:
-      image: fluidcloudnative/juicefsruntime-controller:v0.7.0-e3fde3d
+      image: fluidcloudnative/juicefsruntime-controller:v0.7.0-d06954c
     fuse:
       image: juicedata/juicefs-csi-driver:v0.11.0
 
 webhook:
   enabled: true
-  image: fluidcloudnative/fluid-webhook:v0.7.0-e3fde3d
+  image: fluidcloudnative/fluid-webhook:v0.7.0-d06954c
 

--- a/pkg/ddc/base/runtime_test.go
+++ b/pkg/ddc/base/runtime_test.go
@@ -461,7 +461,7 @@ func TestCleanPolicy(t *testing.T) {
 				// fuse global is set to true since v0.7.0
 				fuse: Fuse{
 					Global:      true,
-					CleanPolicy: v1alpha1.OnDemandCleanPolicy,
+					CleanPolicy: v1alpha1.OnRuntimeDeletedCleanPolicy,
 				},
 			},
 			wantErr: false,
@@ -709,7 +709,7 @@ func TestGetRuntimeInfo(t *testing.T) {
 				// fuse global is set to true since v0.7.0
 				fuse: Fuse{
 					Global:      true,
-					CleanPolicy: v1alpha1.OnDemandCleanPolicy,
+					CleanPolicy: v1alpha1.OnRuntimeDeletedCleanPolicy,
 				},
 			},
 			wantErr: false,

